$OpenBSD: patch-src_main_c,v 1.11 2012/07/06 08:32:13 dcoppa Exp $

Properly initialize libgcrypt
(upstream git commit 96db841a16670b52fa462337c657c6dc6bd2844a)

--- src/main.c.orig	Fri Jul  6 09:59:58 2012
+++ src/main.c	Fri Jul  6 10:00:50 2012
@@ -343,6 +343,9 @@ int main (int argc, char **argv) {
 
 	/* init some things */
 	ao_initialize ();
+	gcry_check_version (NULL);
+	gcry_control (GCRYCTL_DISABLE_SECMEM, 0);
+	gcry_control (GCRYCTL_INITIALIZATION_FINISHED, 0);
 	gnutls_global_init ();
 
 	BarSettingsInit (&app.settings);
